src/applications/personalization/dashboard-2/components/Dashboard.jsx

```diff
@@ -115,6 +118,8 @@ const Dashboard = ({
                 My VA
               </h1>
 
+              {showHealthCareError && <HealthCareLoadError />}
```

Comment:
rather than the CSS you've set up to manage the width of this, I think we should be using the column layout classes similar to how we do it on line 125 below.